src/client.ts

```diff
@@ -68,7 +87,11 @@ export async function startWebDriver(env?: string, timeout: number = 5000) {
  * Stop WebDriver
  * @param timeout Timeout in ms, default 3000ms
  */
-export async function stopWebDriver(timeout: number = 5000) {
+export async function stopWebDriver({
+  timeout = 5000
+}: {
+  timeout?: number;
+} = {}) {
```

Comment:
I've also changed those parameters to a configuration object because otherwise `stopWebDriver ` and `startWebDriver` would take different options for configuring the timeout which would be strange in my opinion.